README.md

```diff
@@ -210,6 +207,33 @@ planned. However, we plan to:
   [Jedi](https://jedi.readthedocs.org/en/latest/) for completing in regular
   Python code (separate completion machinery is still needed for our special
   syntax).
+  
+## Dashboarding
+
+### jupyter/enhancement-proposals (now)
+
+* Create an enhancement proposal to define an extensible layout metadata schema within the Jupyter notebook document format. 
+    * Use the jupyter-incubator/dashboards and Anaconda-server/nbpresent metadata as a guide for what needs to be supported.
+
+### jupyter-incubator/dashboards (0.6.0)
+
+* Update the incubator extension to work with the new schema and deprecate support for the older `urth.layout` schema.
+* Submit an enhancement proposal to graduate the project from incubator as a "classic" Jupyter Notebook extension.
+
+### jupyter-js-plugins (>0.16.0)
+
+* Implement a dashboard layout editor/viewer in Jupyter Lab compatible with the layout metadata schema and jupyter-incubator/dashboards use of that schema to layout dashboards.
+* Improve upon the layout authoring experience afforded in the "classic" Notebook by taking advantage of the multi-view capabilities of the Jupyter Lab interface.
+
+### jupyter/notebook (>4.2)
+
+* Add a URL in the single user notebook server that shows a rendered dashboard.
+    * TODO: This is a straight copy/paste from the hackpad. Why / how did we say this would go into notebook server? The layout mechanism is one extension for classic and a separate plugin for Jupyter Lab. how would this server endpoint know what to serve up to render the dashboard? Or is this a completely separate renderer that has to be written (and why?)
```

Comment:
@ellisonbg @fperez @bollwyvl There's an action item to add a URL to the notebook server. I'm not clear on how that would work given there would be two implementations of the dashboard authoring / rendering in the notebook server, one as a classic extension and one in Jupyter Lab. Which would render the output at the URL? Do we want the server dependent on one or the other since they're both, technically, plug-ins?